# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/TextRange.h
# Commit: 5e92b9c7d479
# Full Hash: 5e92b9c7d4794af1d7ab0fb5c278c2e681a18fc6
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2023-01-28 21:11:06
# Regressor Bug: 1811092
# File Overlap Count: 1
# Description:
#   Bug 1811092 - P1: Add new async add/set text selection message. r=Jamie
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D167201
# ==============================================================================

diff -r 7cebaa85f4ff -r 5e92b9c7d479 accessible/base/TextRange.h
--- a/accessible/base/TextRange.h	Fri Jan 27 20:03:06 2023 +0000
+++ b/accessible/base/TextRange.h	Fri Jan 27 20:15:11 2023 +0000
@@ -111,8 +111,6 @@
    */
   bool Crop(Accessible* aContainer);
 
-  MOZ_CAN_RUN_SCRIPT bool SetSelectionAt(int32_t aSelectionNum) const;
-
   /**
    * Scroll the text range into view.
    */